app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (VaccineError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
It is a good practice to use type hinting for all function arguments and the return type. Here is an example of how you can do it for the 'cafe' argument: 

```python
from typing import List

def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
    ...
```